attributes/barbican.rb

```diff
@@ -35,3 +35,61 @@
   conf['keystone_authtoken']['user_domain_name'] = 'Default'
   conf['keystone_authtoken']['project_domain_name'] = 'Default'
 end
+
```

Comment:
You might want to move these to an extra attributes/openstack-common_attributes.rb file to make the cleanup lateron easier.

Reply:
Good idea. Done.